stream_alert_cli/terraform_generate.py

```diff
@@ -186,25 +226,49 @@ def generate_cluster(**kwargs):
             'alert_processor_vpc_security_group_ids': vpc_config['security_group_ids']
         })
 
-    if modules['cloudwatch_monitoring']['enabled']:
-        # CloudWatch monitoring module
-        cluster_dict['module']['cloudwatch_monitoring_{}'.format(cluster_name)] = {
-            'source': 'modules/tf_stream_alert_monitoring',
-            'sns_topic_arn': '${{module.stream_alert_{}.sns_topic_arn}}'.format(cluster_name),
-            'lambda_functions': [
-                '{}_{}_streamalert_rule_processor'.format(prefix, cluster_name),
-                '{}_{}_streamalert_alert_processor'.format(prefix, cluster_name)
-            ],
-            'kinesis_stream': '{}_{}_stream_alert_kinesis'.format(prefix, cluster_name)
-        }
 
-    # Kinesis module
+def generate_cloudwatch_monitoring(cluster_name, cluster_dict, config):
+    """Add the CloudWatch Monitoring module to the Terraform cluster dict.
+
+    Args:
+        cluster_name [string]: The name of the currently generating cluster
+        cluster_dict [defaultdict]: The dict containing all Terraform config for
+                                    a given cluster.
+        config [dict]: The loaded config from the 'conf/' directory
+    """
+    prefix = config['global']['account']['prefix']
+    cluster_dict['module']['cloudwatch_monitoring_{}'.format(cluster_name)] = {
+        'source': 'modules/tf_stream_alert_monitoring',
+        'sns_topic_arn': '${{module.stream_alert_{}.sns_topic_arn}}'.format(cluster_name),
+        'lambda_functions': [
+            '{}_{}_streamalert_rule_processor'.format(prefix, cluster_name),
+            '{}_{}_streamalert_alert_processor'.format(prefix, cluster_name)
+        ],
+        'kinesis_stream': '{}_{}_stream_alert_kinesis'.format(prefix, cluster_name)
+    }
+
+
+def generate_kinesis(cluster_name, cluster_dict, config):
+    """Add the Kinesis module to the Terraform cluster dict.
+
+    Args:
+        cluster_name [string]: The name of the currently generating cluster
+        cluster_dict [defaultdict]: The dict containing all Terraform config for
+                                    a given cluster.
+        config [dict]: The loaded config from the 'conf/' directory
+    """
+    logging_bucket = '{}.streamalert.s3-logging'.format(
+        config['global']['account']['prefix'])
+    firehose_suffix = config['clusters'][cluster_name]['modules']['kinesis']['firehose']['s3_bucket_suffix']
```

Comment:
V long line. make this something like:
```
firehose_suffix = config['clusters'][cluster_name]['modules'] \ 
    ['kinesis']['firehose']['s3_bucket_suffix']
```

There are a few others like this pylint picks up.